tests/utils.py

```diff
@@ -71,3 +73,33 @@ def skip_if_destructive_and_sensitive(request, base_url):
 def shorten_tag(tag):
     """Returns the short version of a git tag when given the long (or short) version."""
     return tag.split('-')[0]
+
+
+def get_window_size(self, dimension=None):
```

Comment:
functions outside of a class with a reference to self are not good. You'll need to place these within a class. You could add them to the base page and they would be available everywhere.